src/include/pbs_ifl.h

```diff
@@ -581,51 +581,182 @@ DECLDIR char *__pbs_server_location(void);
+
+DECLDIR int pbs_manager(int, int, int, char *, struct attropl *, char *);
+
+DECLDIR int pbs_movejob(int, char *, char *, char *);
+
+DECLDIR int pbs_msgjob(int, char *, int, char *, char *);
 
 DECLDIR int pbs_relnodesjob(int, char *, char *, char *);
+
+DECLDIR int pbs_orderjob(int, char *, char *, char *);
+
+DECLDIR int pbs_rerunjob(int, char *, char *);
+
+DECLDIR int pbs_rlsjob(int, char *, char *, char *);
+
+DECLDIR int pbs_runjob(int, char *, char *, char *);
+
+DECLDIR char **pbs_selectjob(int, struct attropl *, char *);
+
+DECLDIR int pbs_sigjob(int, char *, char *, char *);
+
+DECLDIR void pbs_statfree(struct batch_status *);
+
+DECLDIR struct batch_status *pbs_statrsc(int, char *, struct attrl *, char *);
+
+DECLDIR struct batch_status *pbs_statjob(int, char *, struct attrl *, char *);
+
+DECLDIR struct batch_status *pbs_selstat(int, struct attropl *, struct attrl *, char *);
+
+DECLDIR struct batch_status *pbs_statque(int, char *, struct attrl *, char *);
+
+DECLDIR struct batch_status *pbs_statserver(int, struct attrl *, char *);
+
+DECLDIR struct batch_status *pbs_statsched(int, struct attrl *, char *);
+
+DECLDIR struct batch_status *pbs_stathost(int, char *, struct attrl *, char *);
+
+DECLDIR struct batch_status *pbs_statnode(int, char *, struct attrl *, char *);
+
+DECLDIR struct batch_status *pbs_statvnode(int, char *, struct attrl *, char *);
+
+DECLDIR struct batch_status *pbs_statresv(int, char *, struct attrl *, char *);
+
+DECLDIR struct batch_status *pbs_stathook(int , char *, struct attrl *, char *);
+
+DECLDIR struct ecl_attribute_errors * pbs_get_attributes_in_error(int);
+
+DECLDIR char *pbs_submit(int, struct attropl *, char *, char *, char *);
+
+DECLDIR char *pbs_submit_resv(int, struct attropl *, char *);
+
+DECLDIR int pbs_delresv(int, char *, char *);
+
+DECLDIR int pbs_terminate(int, int, char *);
+
 DECLDIR char *pbs_modify_resv(int, char*, struct attropl *, char *);
 
+DECLDIR preempt_job_info *pbs_preempt_jobs(int, char **);
 #else
-/* Function pointers for IFL calls */
+
+#ifndef __PBS_ERRNO
+#define __PBS_ERRNO
+extern int *__pbs_errno_location(void);
+#define pbs_errno (*__pbs_errno_location ())
+#endif
+
+/* see pbs_connect(3B)			      */
+#ifndef __PBS_SERVER
+#define __PBS_SERVER
+extern char *__pbs_server_location(void);
+#define pbs_server (__pbs_server_location ())
+#endif
+
+extern int pbs_asyrunjob(int, char *, char *, char *);
+
+extern int pbs_alterjob(int, char *, struct attrl *, char *);
+
+extern int pbs_confirmresv(int, char *, char *, unsigned long, char *);
+
+extern int pbs_connect(char *);
+
+extern int pbs_connect_extend(char *, char *);
+
+extern int pbs_disconnect(int);
+
+extern int pbs_manager(int, int, int, char *, struct attropl *, char *);
+
+extern char *pbs_default(void);
+
+extern int pbs_deljob(int, char *, char *);
+
+extern char *pbs_geterrmsg(int);
+
+extern int pbs_holdjob(int, char *, char *, char *);
+
+extern int pbs_loadconf(int);
```

Comment:
Check but I think on the OSS include/ifl_internal.h, there's also a similar extern declaration to pbs_loadconf()...